package.json

```diff
@@ -64,13 +64,13 @@
     "flow-copy-source": "1.2.1",
     "jest": "20.0.4",
     "raf-stub": "2.0.1",
-    "react": "15.6.1",
```

Comment:
Can we please keep the dev dependency version the same and just change the peerDependency version?

Reply:
It's good practice to use same version in develop. So you can quickly find breaking changes. But, it isn't problem for me. I've done it.